### ISSUE TITLE
A little question

### ISSUE THREAD

**Issue by angel-yi** [primepake/better_wav2lip#105]
Hello, thank you very much for sharing the project
I pulled the latest code and trained it according to your training steps
I found that
The current size of the image is 384, not 288
The syncnet and wav2lip models have v2 versions, and I couldn't find any relevant documentation to explain the differences between these two models. I am training HQ_ Wav2lip_ Sam_ When training. py, the import model was modified and it was found that it could not be trained
The image I trained according to 384 has many different colored light spots on the edge or lip area of the image, but the overall resolution is relatively high and realistic
hope
![image](https://github.com/primepake/wav2lip_288x288/assets/51775756/0f91c459-41d1-4670-a0ac-8cafdf82d72b)

**Comment by kunyao2015** [comment]
same question

**Comment by einsqing** [comment]
How long have you been training?

**Comment by angel-yi** [comment]
> 您训练多久了？牙齿修复效果如何？@angel-yi
我训练了几个小时，我的数据集不大，看到有光点效果就停了，牙齿就是下面这个效果
![image](https://github.com/primepake/wav2lip_288x288/assets/51775756/abff8604-3415-4559-a6e7-b5252c2a2448)

**Comment by angel-yi** [comment]
> How long have you been training?How effective is tooth restoration? @angel-yi

I trained for a few hours, and my dataset is not large. When I saw the spot effect, it stopped. The teeth are like the following effect

**Comment by shahidmuneer** [comment]
You need to train for at least 3 lac iterations.

**Comment by shahidmuneer** [comment]
You need a bit higher dataset and train the network until you get fake and real loss, for me after 30 thousand iterations. I am not getting the above spots.

**Comment by angel-yi** [comment]
> You need a bit higher dataset and train the network until you get fake and real loss, for me after 30 thousand iterations. I am not getting the above spots.

Thank you, I am already increasing the size of the dataset and hope to improve this situation

**Comment by primepake** [comment]
you should FID metric to monitor which is best model and I think you should continue train when the loss is sideways

**Comment by lililuya** [comment]
Hi,
I am a little confused about the size of 288 and 384, the train_syncnet_sam.py  imports the "SyncNet_color_384", which is from the syncnet.py module. But the hq _wav2lip_sam_train.py imports the "SyncNet_color",which is from the syncnetv2.py module. I compare the two network and it's different. Is there i miss something important? I am looking for your reply! Thanks!

**Comment by primepake** [comment]
Hi you can check the syncnet should be 384, I'm too lazy to update so all the training should be in 384

**Comment by angel-yi** [comment]
> you should FID metric to monitor which is best model and I think you should continue train when the loss is sideways

Thank you, I am constantly trying to train

**Comment by huangxin168** [comment]
> Hi, I am a little confused about the size of 288 and 384, the train_syncnet_sam.py imports the "SyncNet_color_384", which is from the syncnet.py module. But the hq _wav2lip_sam_train.py imports the "SyncNet_color",which is from the syncnetv2.py module. I compare the two network and it's different. Is there i miss something important? I am looking for your reply! Thanks!

I also confused on this.
1. SyncNet_color_384 from the syncnet.py module
2. SyncNet_color from the syncnetv2.py module
1 and 2 are different modules

so in file hq _wav2lip_sam_train.py, can I modify as follow?
# from models import SyncNet_color as SyncNet
from models import SyncNet_color_384 as SyncNet

**Comment by 1059692261** [comment]
> > you should FID metric to monitor which is best model and I think you should continue train when the loss is sideways
> 
> Thank you, I am constantly trying to train

Have you achieve any success in training? I succeed in training syncnet, the loss has decreased to 0.25. However in wav2lip training, the sync loss remains at a large value around 4, and the sample images show that model just copy the mouth part of the reference image to the predict one.

**Comment by lililuya** [comment]
@huangxin168 Yes, u're right!

**Comment by kike-0304** [comment]
> > > 你应该使用 FID 指标来监控哪个是最好的模型，我认为当损失横向时你应该继续训练
> > 
> > 
> > 谢谢，我一直在努力训练
> 
> 您在培训中取得了成功吗？我成功训练了syncnet，损失减少到了0.25。然而，在 wav2lip 训练中，同步损失仍然保持在 4 左右的较大值，并且样本图像显示模型只是将参考图像的嘴部复制到预测图像。



> > > you should FID metric to monitor which is best model and I think you should continue train when the loss is sideways
> > 
> > 
> > Thank you, I am constantly trying to train
> 
> Have you achieve any success in training? I succeed in training syncnet, the loss has decreased to 0.25. However in wav2lip training, the sync loss remains at a large value around 4, and the sample images show that model just copy the mouth part of the reference image to the predict one.

I have the same problem. After training, the inferred image is the same as the original image, and it seems that the audio did not play a role in driving the mouth.
